### PR TITLE
New package: pfff

### PR DESCRIPTION
**Pull request by jprjr**
This depends on  #2381 and  #2382

### CODE REVIEW

**Comment by chneukirchen** [comment]
I use opam to install all ocaml tools. pfff is there too, ymmv.

File: srcpkgs/pfff/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+checksum="d31c68f1ebb1770297eb26441d21d83728c0fbdb9c7bb3be27625db45fae72d1"
+makedepends="perl ocaml ocaml-findlib camlp4 zlib-devel ncurses-devel"
+depends="ocaml"
+nocross="yes"
+
+do_build() {
+	make clean
+	make depend
+	make
+	make opt
```
Comment (Gottox) [inline srcpkgs/pfff/template]: Why 4 make calls?

Comment (jprjr) [inline srcpkgs/pfff/template]: You're right, the 'make clean' isn't needed. Their official docs list three make calls, though: https://github.com/facebook/pfff/wiki/Main

**Comment by Vaelatern** [comment]
Please make some change and re-push since it seems at the time of last push, Travis was unable to find `ocaml-findlib` and thus failed on non-arm builds. This is, I believe, no longer an issue.

**Comment by jprjr** [comment]
Done, I just rebased from master and pushed.

**Comment by xtraeme** [comment]
AFAIK ocaml binaries must not be stripped, probably you should set `nostrip=yes`.

**Pull request merged**